### DISCUSSION TITLE
House Load

### DISCUSSION THREAD

**Discussion by gjhill6100** [nathanmarlor/foxess_modbus#301] [Q&A]
Hi,

What is the correct sensor to monitor to show actual current house load ?

I have a H1-5 and also a S2500 which is monitored using CT2 Meter in to the H1-5.

If I try to use Load Power, then this does not give the correct current house load power

Thanks

**Comment by canton7** [comment]
So the H1 and the S2500 are in parallel, both feeding energy into the house?

I suspect you'll need to create a template sensor which calculates your load as the power which both inverters are outputting minus the power going to the grid, ie Inverter Power + CT2 - Grid CT.

**Reply by canton7** [reply]
Actually, since load should just be Inverter Power - Grid CT, you should be able to do Load + CT2?

**Comment by delibury** [comment]
I use load_power + ct2_meter.
I've got an AC1 plus a separate solar inverter.

**Comment by delibury** [comment]
This is my template sensor 

 '- name: "PV Power"
    unique_id: pv_power
    unit_of_measurement: 'kW'
    device_class: power
    state: >
      {{ [0, (states('sensor.ct2_meter') | float(default=0) * 1)]|max }}'
             

  '- name: "House Load Power"
    unique_id:  house-load-power
    unit_of_measurement: 'kW'
    device_class: power
    state: >
      {% set load = states('sensor.load_power') | float(default=0) %}
      {% set pv = states('sensor.pv_power') | float(default=0) %}
      {% set power = ( load + pv ) %}
      {{ "{:.3f}".format(power) }}'
